src/backend/tests/test_crawler_expunger.py

```diff
@@ -254,3 +257,72 @@ def test_expunger_for_record_with_mj_over_21(record_with_mj_over_21):
             status=EligibilityStatus.ELIGIBLE, reason="Eligible now", date_will_be_eligible=date(2001, 3, 3)
         )
     }
+
+@pytest.fixture
+def record_with_mj_under_21_and_traffic_violation():
+    return CrawlerFactory.create(
+        record=YoungDoe.SINGLE_CASE_RECORD,
+        cases={"CASEJD1": CaseDetails.CASE_MJ_AND_TRAFFIC_CONVICTION,})
+
+def test_expunger_for_record_with_mj_under_21_not_blocked_by_traffic(record_with_mj_under_21_and_traffic_violation):
+    expunger_result = Expunger.run(record_with_mj_under_21_and_traffic_violation)
+
+    assert isinstance(record_with_mj_under_21_and_traffic_violation.charges[0].charge_type, MarijuanaUnder21)
+    assert isinstance(record_with_mj_under_21_and_traffic_violation.charges[1].charge_type, TrafficViolation)
+
+    assert expunger_result["CASEJD1-1"] == TimeEligibility(status=EligibilityStatus.ELIGIBLE,
+                                                           reason="Eligible now",
+                                                           date_will_be_eligible=date(1999, 3, 3)) #one year after conviction
```

Comment:
```suggestion
                                                           date(1998, 3, 3)+relativedelta(years=1)
```

To be consistent with what you did below

Reply:
Even better would be to just use `charge.disposition.date + relativedelta(years=1)`